### PR TITLE
Add Game Events Tutorial

### PR DESCRIPTION
**Pull request by CommanderFoo**
First draft of the game events tutorial.

### CODE REVIEW

File: src/tutorials/game_events.md [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+    - Tutorial
+---
+
+# Game Events
+
+---
+
+## Overview
+
+In this tutorial you will learn how to award double coins when objects are killed while the event is active. After the event has ended, players will earn coins at the standard rate.
```
Comment (coreslinkous) [inline src/tutorials/game_events.md:15]: First sentence about what Game Events is and link to the ref right away.

File: src/tutorials/game_events.md [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+    </video>
+</div>
+
+## Summary
+
+Game Events can be used for a wide range of things for your game to encourage players to come back and play. The game events don't need to be tied to double resources, like XP, coins. There could be in world events where a special enemy spawns only when the event is active, and drops special loot for players.
+
+## Learn More
+
+[Perks Program](/references/perks/program.md) | [Game Events Reference](/references/game_events.md) | [Damageable Objects](/references/damageable_objects.md)
```
Comment (coreslinkous) [inline src/tutorials/game_events.md]: Game Events first

**Review by StanzillaManticore — APPROVED** [review]

**Review by coreslinkous — COMMENTED** [review]
Otherwise great. 

- Knowledge Level: None doesn't quite sit right with me, but anything else might alienate the folks who could benefit from this. Maybe rename to Prerequisite Skills or something.

- Would be great if you could link to the API Type/Namespace each time you use a new one in the description of the step.

File: src/tutorials/game_events.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
```
Comment (StanzillaManticore) [inline src/tutorials/game_events.md]:
```suggestion
[Perks Program](../references/perks/program.md) | [Game Events Reference](../references/game_events.md) | [Damageable Objects](../references/damageable_objects.md)
```

**Pull request merged**